applications/FluidDynamicsApplication/custom_processes/distance_modification_process.h

```diff
@@ -69,10 +69,10 @@ class KRATOS_API(FLUID_DYNAMICS_APPLICATION) DistanceModificationProcess : publi
 
     /// Constructor.
     DistanceModificationProcess(
-        ModelPart& rModelPart, 
+        ModelPart& rModelPart,
         const double FactorCoeff, //TODO: Remove it (here for legacy reasons)
         const double DistanceThreshold,
-        const bool CheckAtEachStep, 
+        const bool CheckAtEachStep,
         const bool NegElemDeactivation,
         const bool RecoverOriginalDistance);
```

Comment:
If this constructor is only here for legacy reasons, could you add a deprecation warning for it? Either using the macro here or by printing a warning when it is instantiated through python.

Reply:
Just the FactorCoeff variable is legacy (I decided to keep it in case it is needed to run an old case). I still use this constructor sometimes.